### DISCUSSION TITLE
Cask RsyncUI has a checksum mismatch

### DISCUSSION THREAD

**Discussion by DaveMcCloud** [Homebrew/discussions#4958] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.24-12-gc32bd1c
ORIGIN: https://github.com/Homebrew/brew
HEAD: c32bd1c7cc0c3b1b914845bddfeda53f4d877a3f
Last commit: 11 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 046efe69c82ddedbbaf85af72c03527ca315c534
Core tap last commit: 4 weeks ago
Core tap branch: master
Core tap JSON: 08 Dec 08:52 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 12
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: dodeca-core 64-bit westmere
Clang: 15.0.0 build 1500
Git: 2.15.1 => /usr/local/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 14.1.2-x86_64
CLT: 15.0.0.0.1.1694021235
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

The checksum for RsyncUI has changed

Error: rsyncui: SHA256 mismatch
Expected: 49ef640c14c504030c4552a0551b37be35b7c1c9d4be9650b47b4b0cfec1148a
  Actual: f84c2cac8a2eb47ac75712c36adee047e66dd18a515aa58a34213840ccdee8fc
    File: /Volumes/Home/dave/Library/Caches/Homebrew/downloads/0864dbe0dd64289d9529132163846d0dd8b9e913d793cbe38cf2edef8f4d2031--RsyncUI.1.7.9.dmg
To retry an incomplete download, remove the file above.

I can see from https://github.com/rsyncOSX/RsyncUI/releases/ that the file was updated last night after the change to Casks/r/rsyncui.rb

### Relevant casks

RsyncUI

**Comment by chenrui333** [comment, marked as answer]
resolved by https://github.com/Homebrew/homebrew-cask/pull/162051
